packages/db-kit/package.json

```
@@ -0,0 +1,75 @@
+{
+  "name": "@truffle/db-kit",
+  "version": "0.1.0",
+  "description": "Utility belt for Truffle and @truffle/db",
+  "keywords": [
+    "smart-contract",
+    "truffle"
+  ],
+  "author": "g. nicholas d'andrea <gnidan@trufflesuite.com>",
+  "homepage": "https://github.com/trufflesuite/truffle#readme",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "directories": {
+    "dist": "dist"
+  },
+  "bin": {
+    "db-kit": "./dist/bin/cli.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trufflesuite/truffle.git"
+  },
```

If you add a `directory` then npm will link directly to this package's directory instead of the root.

```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/trufflesuite/truffle.git",
    "directory": "packages/db-kit"
  },
```

---

Neat!